src/db/db/gsiDeclDbLibrary.cc

```diff
@@ -897,7 +955,10 @@ Class<db::PCellParameterDeclaration> decl_PCellParameterDeclaration ("db", "PCel
   gsi::method ("TypeLayer", &pd_type_layer, "@brief Type code: a layer (a \\LayerInfo object)") +
   gsi::method ("TypeShape", &pd_type_shape, "@brief Type code: a guiding shape (Box, Edge, Point, Polygon or Path)") +
   gsi::method ("TypeCallback", &pd_type_callback, "@brief Type code: a button triggering a callback\n\nThis code has been introduced in version 0.28.") +
-  gsi::method ("TypeNone", &pd_type_none, "@brief Type code: unspecific type")
+  gsi::method ("TypeNone", &pd_type_none, "@brief Type code: unspecific type") +
+  gsi::method ("REJECT", &pd_action_reject, "@brief Action type: reject violating parameter") +
```

Comment:
In other places, constants are not all upper case, so it would be more lik "Reject", "Accept" ... but anyway, I don't understand why we should have anything else than "Reject": so either a value is limited or not.

Reply:
Will be adjusted.
Accept coming from PyCell-API, e.g. for test-reason during development.